docs/reference/data-streams/downsampling-manual.asciidoc

```diff
@@ -0,0 +1,450 @@
+[[downsampling-manual]]
+=== Run downsampling manually
+++++
+<titleabbrev>Run downsampling manually</titleabbrev>
+++++
+
+preview::[]
+
+This is a simplified example that allows you to see quickly how <<downsampling,downsampling>> works to reduce the storage size of a sampled set of metrics. The example uses typical Kubernetes system monitoring data. To test out downsampling, follow these steps:
```

Comment:
```suggestion
This is a simplified example that allows you to see quickly how <<downsampling,downsampling>> works to reduce the storage size of a time series index. The example uses typical Kubernetes cluster monitoring data. To test out downsampling, follow these steps:
```

Reply:
Fixed in [507419d](https://github.com/elastic/elasticsearch/pull/88571/commits/507419dcc6e4498ad50f37505d150e5de4db8a77)